tests/integration/sync/test_sync_adl.py

```diff
@@ -147,7 +147,7 @@ def test_sync_watch_code(self):
         )
         read_until_string(
             self.watch_process,
-            "\x1b[32mFinished syncing Function Layer Reference Sync HelloWorldFunction.\x1b[0m\n",
+            "\x1b[32mFinished syncing Layer HelloWorldFunction",
```

Comment:
Let's change this to function instead of layer, and update L136 to be the same

Reply:
Sorry nevermind this is actually supposed to be layer, but I'm raising a CR to update the validation on L136 to be the same as this one